extra-tests/tests/LogHandler_Tests_Flow.cls

```diff
@@ -8,16 +8,16 @@ private class LogHandler_Tests_Flow {
     @IsTest
     static void runFlowPluginWhenConfigured() {
         // Assumption: the Flow LogHandler_Tests_Flow makes an update to the current user's FirstName
-        // The specific action within the Flow isn't that important - we just want to make sure...
-        // ...that that Flow is dynamically executed
+        // The specific action within the Flow isn't that important - we just want to make sure
+        // that that Flow is dynamically executed
         String pluginFlowApiName = 'LogHandler_Tests_Flow';
         String expectedUserFirstName = 'Logger-Flow-Test';
         System.assertNotEquals(expectedUserFirstName, UserInfo.getFirstName(), 'Current user first name did not match expected user first name.');
 
         Test.startTest();
 
         // Use the mock configurations
-        LoggerSObjectHandlerPlugin__mdt plugin = new LoggerSObjectHandlerPlugin__mdt(PluginType__c = 'Flow', PluginApiName__c = pluginFlowApiName);
+        LoggerPlugin__mdt plugin = new LoggerPlugin__mdt(PluginType__c = 'Flow', PluginApiName__c = pluginFlowApiName);
```

Comment:
having _also_ just made this style of change, I really like the updated CMDT names

Reply:
Yeah, the old name was descriptive, but just so. darn. loooooooooong. I like the shorter name - it's more concise, but still feels descriptive.